File: LMS_CMS_Frontend/src/app/Models/LMS/session-group-dto.ts
import { SubjectTeacherDTO } from "./subject-teacher-dto";

export class SessionGroupDTO {
      constructor(
       public sessionId: number=0,
       public dutyTeacherId: number=0,
        public dutyTeacherName: string = '',
       public subjects: SubjectTeacherDTO[]=[],
    ) { }
}

File: LMS_CMS_Backend/LMS_CMS_BL/DTO/LMS/SessionGroupDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LMS_CMS_BL.DTO.LMS
{
    public class SessionGroupDTO
    {
        public long SessionId { get; set; }
        public int PeriodIndex { get; set; }
        public long? DutyTeacherId { get; set; }
        public string DutyTeacherName { get; set; }
        public List<SubjectTeacherDTO> Subjects { get; set; }

    }
}
